content/en/blog/_posts/2021-04-15-Three-Tenancy-Models-for-Kubernetes.md

```diff
@@ -0,0 +1,81 @@
+
+## Clusters as a Service
+
+With the *clusters-as-a-service* usage model, each tenant gets their own cluster.  This model allows tenants to have different versions of cluster-wide resources such as CRDs, and provides full isolation of the Kubernetes control plane.
+
+The tenant clusters may be provisioned using projects like [Cluster API (CAPI)](https://cluster-api.sigs.k8s.io/) where a management cluster is used to provision multiple workload clusters. A workload cluster is assigned to a tenant and tenants have full control over cluster resources. Note that in most enterprises a central platform team may be responsible for managing required add-on services such as security and monitoring services, and for providing cluster lifecycle management services such as patching and upgrades. A tenant administrator may be restricted from modifying the centrally managed services and other critical cluster information. 
+
+## Control planes as a Service
+
+In a variation of the *clusters-as-a-service* model, the tenant cluster may be a **virtual cluster** where each tenant gets their own dedicated Kubernetes control plane but share worker node resources. As with other forms of virtualization, users of a virtual cluster see no significant differences between a virtual cluster and other Kubernetes clusters. This is sometimes referred to as `Control Planes as a Service` (CPaaS).
+
+A virtual cluster of this type shares worker node resources and workload state independent control plane components, like the scheduler. Other workload aware control-plane components, like the API server, are created on a per-tenant basis to allow overlaps, and additional components are used to synchronize and manage state across the per-tenant control plane and the underlying shared cluster resources. With this model users can manage their own cluster-wide resources. 
+
+The [Virtual Cluster](https://github.com/kubernetes-sigs/multi-tenancy/tree/master/incubator/virtualcluster) project implements this model, where a `supercluster` is shared by multiple `virtual clusters`. The [Cluster API Nested](https://github.com/kubernetes-sigs/cluster-api-provider-nested) project is extending this work to conform to the CAPI model, allowing use of familiar API resources to provision and manage virtual clusters.
+
+## Security considerations
+
+Cloud native security involves different system layers and lifecycle phases as described in the [Cloud Native Security Whitepaper](https://www.cncf.io/blog/2020/11/18/announcing-the-cloud-native-security-white-paper/) from CNCF SIG Security.  Without proper security measures implemented across all layers and phases, Kubernetes tenant isolation can be compromised and a security breach with one tenant can threaten other tenants.
```

Comment:
Would it make more sense to link the kubernetes blog post instead of cncf blog post, for the cloud native security whitepaper: https://kubernetes.io/blog/2020/11/18/cloud-native-security-for-your-clusters/

Reply:
I'm fine with either link. @PushkarJ the article isn't live so you _might_ have time to propose a revision and get it reviewed.

My gut feeling is that, actually, for only a small tweak it is not worth the effort.

Reply:
To close the loop here, PR #27555 updates the above link for whitepaper